src/test/java/com/google/api/codegen/testdata/python_doc_message_library.baseline

```diff
@@ -18,7 +18,7 @@ class FieldMask(object):
     """
 
     Attributes:
-      materials (list[enum :class:`google.cloud.gapic.example.library.v1.enums.FieldMask.Material`])
+      materials (list[enum :class:`google.cloud.library_v1.gapic.enums.FieldMask.Material`])
```

Comment:
Is this a valid docstring class identifier? I rather doubt it. I think the `enum ` prefix is not valid.